### PR TITLE
Add Horizontal Rule component

### LINKED ISSUE #53: Put HorizontalRule component into Psammead
**Is your feature request related to a problem? Please describe.**
Blocked on [#1033](https://github.com/BBC-News/simorgh/pull/1033) - HorizontalRule component uses Text component in its Storybook render. The linked PR changes usage from Text to Paragraph.

Related to https://github.com/BBC-News/simorgh/issues/852 - one needs to happen before the other - doesn't matter which.

Corresponding 'pull from Psammead' issue in Simorgh: https://github.com/BBC-News/simorgh/issues/998.

**Describe the solution you'd like**
A PR in Psammead, which adds HorizontalRule component.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Unit tests (if any) will be transferred from Simorgh.

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #53

Creates the `@bbc/psammead-horizontal-rule` component. Originally in [Simorgh](https://github.com/BBC-News/simorgh)

- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by sareh** [comment]
The unit tests pass, but since Storybook on `latest` is broken, the component can't be viewed yet.

**Review by dr3 — APPROVED** [review]

File: packages/components/psammead-horizontal-rule/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Psammead Horizontal Rule Changelog
+
+| Version | Description |
+|---------|-------------|
+| 0.1.0   | [PR#](https://github.com/BBC-News/psammead/pull/) Create initial package with Horizontal Rule component pulled in from [Simorgh](https://github.com/BBC-News/simorgh). |
```
Comment (ChrisBAshton) [inline packages/components/psammead-horizontal-rule/CHANGELOG.md:5]:
```suggestion
| 0.1.0   | [PR#122](https://github.com/BBC-News/psammead/pull/122) Create initial package with Horizontal Rule component pulled in from [Simorgh](https://github.com/BBC-News/simorgh). |
```

File: packages/components/psammead-horizontal-rule/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Example usage with Psammead Paragraph:
+
+```jsx
+import HorizontalRule from '@bbc/psammead-horizontal-rule';
+import Paragraph from '@bbc/psammead-paragraph';
+
+const WrappingComponent = () => (
+  <Paragraph>Text here.</Paragraph>
+    <HorizontalRule/>
```
Comment (ChrisBAshton) [inline packages/components/psammead-horizontal-rule/README.md:25]:
```suggestion
  <HorizontalRule />
```

(nesting seems off)

File: packages/components/psammead-horizontal-rule/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+```
+
+Example usage with standard paragraph elements:
+
+```jsx
+import HorizontalRule from '@bbc/psammead-horizontal-rule';
+
+const WrappingComponent = () => (
+  <p>Text here.</p>
+    <HorizontalRule/>
```
Comment (ChrisBAshton) [inline packages/components/psammead-horizontal-rule/README.md:37]:
```suggestion
  <HorizontalRule />
```

(nesting seems off)

**Review by ChrisBAshton — CHANGES_REQUESTED** [review]
Changelog.md and Readme tweaks please 👍 otherwise looks great!

**Comment by sareh** [comment]
Closed as we're not using Horizontal Rule anymore.

**Pull request closed without merging**